src/PeriodCollection.php

```diff
@@ -144,6 +144,21 @@ public function isEmpty(): bool
         return count($this->periods) === 0;
     }
 
+    public function subtract(PeriodCollection|Period $periods)
+    {
+        if ($periods instanceof Period) {
+            $periods = new self($periods);
```

Comment:
Change to `new static()` so that it also works if someone extends from the `PeriodCollection` class